flutter_keyboard_visibility_linux/pubspec.yaml

```diff
@@ -16,7 +18,11 @@ flutter:
         dartPluginClass: FlutterKeyboardVisibilityPluginLinux
 
 dependencies:
-  flutter_keyboard_visibility_platform_interface: ^2.0.0
+  flutter_keyboard_visibility_platform_interface:
+    git:
+      url: https://github.com/HaltiaAI/flutter_keyboard_visibility.git
```

Comment:
the reviewers have the ability to test it via current pubspec dependencies.
After I get an approval for this PR, I will revert pubspec changes

Reply:
also, if you are satisfied with general idea, I will update all places of the doc comments across the package to the `status` instead of `isKeyboardVisible`

Reply:
@MisterJimson